### PR TITLE
Update TooWaBlue theme (2.61) & Deutsch locale

### PR DESCRIPTION
**Pull request by TooWaBoo**

### CODE REVIEW

File: rtdata/languages/Deutsch [hunk 1]
```diff
@@ -2226,25 +2227,25 @@ ZOOMPANEL_ZOOMOUT;Herauszoomen\nTaste: <b>-</b>
...
+HISTORY_MSG_297;(Rauschreduzierung)\nQualität
+HISTORY_MSG_METADATA_MODE;(Metadaten)\nKopiermodus
+PREFERENCES_CROP;Einstelleung des Ausschnittswerkzeuges
+PREFERENCES_CROP_AUTO_FIT;Automatischer Zoom des Ausschnitts
+PREFERENCES_CROP_GUIDES;Hilfslinien anzeigen wenn Ausschnitt nicht verändert wird
+PREFERENCES_CROP_GUIDES_FRAME;Rahmen
+PREFERENCES_CROP_GUIDES_FULL;Vorgabe des Ausschnittswerkzeuges
+PREFERENCES_CROP_GUIDES_NONE;Keine
+PREFERENCES_EDITORCMDLINE;Benutzerdefinierte Befehlszeile
+TP_DIRPYRDENOISE_CHROMINANCE_CURVE_TOOLTIP;Erhöht / Reduziert die Intensität der\nChrominanz-Rauschreduzierung in\nAbhängigkeit der Farbsättigung.\n
```
Comment (Beep6581) [inline rtdata/languages/Deutsch]: @TooWaBoo is the trailing `\n` intentional?

Comment (TooWaBoo) [reply rtdata/languages/Deutsch]: Yes it  is. I want an empty line between the text and the hotkey

File: rtdata/languages/Deutsch [hunk 2]
```diff
@@ -2226,25 +2227,25 @@ ZOOMPANEL_ZOOMOUT;Herauszoomen\nTaste: <b>-</b>
...
-!TP_RAW_PIXELSHIFTONEGREEN;Use one green instead of average
-!TP_RAW_PIXELSHIFTONEGREEN_TOOLTIP;Use one green instead of averaging two greens for regions without motion.
+BATCHQUEUE_STARTSTOPHINT;Startet / Stoppt die Verarbeitung\nder Warteschlange.\n\nTaste: <b>Strg</b> + <b>s</b>
+GENERAL_SLIDER;Regler
+HISTORY_MSG_173;(Rauschreduzierung)\nLuminanzdetails
+HISTORY_MSG_203;(Rauschreduzierung)\nFarbraum
+HISTORY_MSG_256;(Rauschreduzierung)\nMedianfilter - Mediantyp
+HISTORY_MSG_297;(Rauschreduzierung)\nQualität
+HISTORY_MSG_METADATA_MODE;(Metadaten)\nKopiermodus
+PREFERENCES_CROP;Einstelleung des Ausschnittswerkzeuges
```
Comment (Floessie) [inline rtdata/languages/Deutsch]: Typo?

**Comment by TooWaBoo** [comment]
@Floessie 
It took me a while to find the typo. :-)
Einstelleung -> Einstellung
@Beep6581 
I've removed the trailing \n

**Pull request merged**